src/net/sourceforge/kolmafia/request/FightRequest.java

```diff
@@ -9554,6 +9560,48 @@ private static void payActionCost(final String responseText) {
         }
         break;
 
+      case SkillPool.RE_PROCESS_MATTER:
+        // NAME launches almost all of its body mass at your foe. The blob
+        // careens off of THEM and then reforms into an identical copy, which
+        // is so surprised to exist that it immediately keels over.
+        if (responseText.contains("launches almost all of its body mass at your foe")) {
+          // This skill adds this monster to list of goose-absorption
+          // It resets the weight of the Grey Goose to 1 lb.
+          KoLCharacter.getFamiliar().setExperience(0);
+        }
+        break;
+
+      case SkillPool.MEATIFY_MATTER:
+        // NAME detaches a big chunk of itself, slaps it onto your opponent's
+        // torso, and atomically reconfigures the whole mess into Meat.
+        if (responseText.contains("reconfigures the whole mess into Meat")) {
+          // This skill can only be used once per day.
+          Preferences.setBoolean("_meatifyMatterUsed", true);
+          KoLCharacter.addAvailableCombatSkill(SkillPool.MEATIFY_MATTER);
```

Comment:
perhaps you meant remove? Despite that, I don't think this is necessary: when the familiar weight is reset to 1, the skill will disappear anyway, right?

Reply:
Right on both accounts.

Reply:
(The thinking was that since it's a once-per day skill, explicitly make it unavailable, rather than having it become unavailable as a side effect of the weight change. But the comment will suffice. :)